docs/content/getting-started.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Quickstart with Bolt for JavaScript
+sidebar_label: Quickstart
+---
+
+This quickstart guide aims to help you get a Slack app using Bolt for JavaScript up and running as soon as possible!
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+When complete, you'll have a local environment configured with a customized [app](https://github.com/slack-samples/bolt-js-getting-started-app) running that responds to a simple greeting.
+
+:::tip[Reference for readers]
+
+In search of the complete guide to building an app from scratch? Check out the [building an app](/building-an-app) guide.
+
+:::
+
+#### Prerequisites
+
+A few tools are needed for the following steps. We recommend using the [**Slack CLI**](https://tools.slack.dev/slack-cli/) for the smoothest experience, but other options remain available.
+
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
+
+Install the latest version of the Slack CLI to get started:
+
+- [Slack CLI for macOS & Linux](https://tools.slack.dev/slack-cli/guides/installing-the-slack-cli-for-mac-and-linux)
+- [Slack CLI for Windows](https://tools.slack.dev/slack-cli/guides/installing-the-slack-cli-for-windows)
+
+Then confirm a successful installation with the following command:
+
+```sh
+$ slack version
+```
+
+An authenticated login is also required if this hasn't been done before:
+
+```sh
+$ slack login
+```
+
+</TabItem>
+<TabItem value="terminal" label="Terminal">
+
+Tooling for the terminal can also be used to follow along:
+
+- [Git](https://git-scm.com/downloads)
+- [Node.js](https://nodejs.org/en/download)
+
+Once installed, make sure recent versions are being used:
+
+```sh
+$ git --version
+$ node --version
+```
+
+</TabItem>
+</Tabs>
+
+:::info[A place to belong]
+
+A workspace where development can happen is also needed.
+
+We recommend using [developer sandboxes](https://docs.slack.dev/tools/developer-sandboxes) to avoid disruptions where real work gets done.
+
+:::
+
+## Creating a project {#creating-a-project}
+
+With the toolchain configured, it's time to set up a new Bolt project. This contains the code that handles logic for your app.
+
+If you don’t already have a project, let’s create a new one!
+
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
+
+A blank template can be used to start with project scaffolding:
+
+```sh
+$ slack create first-bolt-app --template slack-samples/bolt-js-getting-started-app
+$ cd first-bolt-app
+```
+
+After a project is created you'll have a `package.json` file with project details and a `.slack` directory for application use.
```

Comment:
suggestion: I think we could clarify the purpose of each of these files, such as:
```suggestion
After a project is created you'll have a `package.json` file for app dependencies and a `.slack` directory for Slack CLI configuration.
```